integration/angular_cli/e2e/tsconfig.json

```diff
@@ -3,7 +3,7 @@
   "compilerOptions": {
     "outDir": "../out-tsc/e2e",
     "module": "commonjs",
-    "target": "es5",
+    "target": "es2018",
```

Comment:
I assume that this is intentionally es2018 because our oldest node version doesn't support module=es2020. Can you please leave a comment here so that we know in the future? Thanks

Reply:
This is target not module 😜